ipam/client/backends/phpipam.py

```diff
@@ -218,6 +218,54 @@ def add_top_level_subnet(self, subnet, description):
 
         return True
 
+    def add_subnet(self, subnet, parent_subnet, description):
+        """
+        Add a subnet if can be inserted in parent subnet.
+        """
+        subnet_len = subnet.prefixlen
+        children_subnets = list(parent_subnet.subnets(new_prefix=subnet_len))
+        # TODO: use supernet_of or subnet_of (Python 3.7)
+        if subnet not in children_subnets:
```

Comment:
You may check that ip_interface(subnet/parent_mask)'s network address is equal to parent's network address to simplify this check.

Reply:
Please check the latest commits :+1: